### DISCUSSION TITLE
How to work with NewConnectionError after a lot of file savings?

### DISCUSSION THREAD

**Discussion by mihalt** [Terrance/SkPy#236] [Q&A]
I tried too much times msg.fileContent. And get this error after I try to do the same request after 1 hour and more. 

requests.exceptions.ConnectionError: HTTPSConnectionPool(host='cus1-api.asm.skype.com', port=443): Max retries exceeded with url: /v1/objects/0-cus-d2-36227e6dee496fca7d66d3b42b2bc0fd/views/imgpsh_fullsize (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1cffb93f10>: Failed to establish a new connection: [Errno -2] Name or service not known'))

What should I do in this case? To relogin to account?

**Comment by Terrance** [comment]
> I tried too much times msg.fileContent

[Rate limiting](https://skpy.t.allofti.me/background/authentication.html#rate-limiting)

> cus1-api.asm.skype.com

That domain doesn't exist, so either you're looking at an old file on a server that no longer exists, or the server URLs are dynamic and you've cached it for too long.  Fetch the message again rather than caching file URLs and see if the new one is valid.

**Reply by mihalt** [reply]
I did it through `msg.fileContent`. So, a bit not understood, what should I do to renew dinamic url on file in msg object? Should I run getMsgs() again to get the same msg by it's id?
